src/matlab2tikz.m

```diff
@@ -407,9 +407,19 @@ function matlab2tikz(varargin)
 end
 % ==============================================================================
 function fid = fileOpenForWrite(m2t, filename)
-    encoding = switchMatOct({'native', m2t.cmdOpts.Results.encoding}, {});
+    % Set the encoding of the output file.
+    % Currently only MATLAB supports different encodings.
+    fid = -1;
+
+    if strcmpi(getEnvironment, 'MATLAB')
+        fprintf('MATLAB\n')
```

Comment:
whoops... leftover from debugging (it took me some time to notice there are three places that supply this function)